docker-compose.override-example.yml

```diff
@@ -0,0 +1,17 @@
+version: '3.7'
+services:
+  database:
+    environment:
+      MYSQL_ROOT_PASSWORD: root
+      MYSQL_USER: root
+      MYSQL_PASSWORD: root
+      MYSQL_DATABASE: testdb
+  web:
+    volumes:
+      - './server:/app/server'
+    environment:
+      DB_URL: mysql://root:root@database:3306/testdb
```

Comment:
Started to think about would it be more clear so specify host, username, password etc in separate env variables and passing them each to typeorm when you make connection? (docs https://typeorm.io/#/connection) vs specifying them all in one string imo hiding the information 🤔

Reply:
It's a real shame that the url thing doesn't work correctly while making the connection either! The database in the end has to be declared on it's own anyway which alone I thought could be a very reasonable reason for changing back to seperated declaring like it used to be in the beginning! 

But im still a big fan of the url syntax thing and I believe that it's quite widely used also too!? The database provider in Heroku at least provided the credential with just a simple url!

I do however agree that having to use the deglared service as the host address is quite confusing! But would it really be any less confusing if it were declared: `DB_ADDRESS: database` ? Perhaps the real thing that should be renamed is the name of the service?